agent/trufflehog_agent.py

```diff
@@ -42,7 +42,7 @@ def _report_vulnz(self, vulnz: list[dict[str, Any]], message: m.Message) -> None
                 risk_rating=agent_report_vulnerability_mixin.RiskRating.HIGH
                 if vuln["Verified"] is True
                 else agent_report_vulnerability_mixin.RiskRating.POTENTIALLY,
-                technical_detail=f'Secret `{vuln["Redacted"]}` found in file `{message.data.get("path")}`',
+                technical_detail=f'Secret `{vuln["Raw"]}` found in file `{message.data.get("path")}`',
```

Comment:
```suggestion
                technical_detail=f'Secret `{vuln.get("Raw")}` found in file `{message.data.get("path")}`',
```
also if it is none you need to cover this case